_data/sites.json

```
@@ -696,18 +696,18 @@
     },
 
     {
-        "name": "Battle.net",
-        "url": "https://eu.battle.net/support/en/ticket/submit",
-        "difficulty": "hard",
-        "notes": "Customer Support will require you to send a signed written letter confirming your wishes, your account details and a copy of legal identification (passport, drivers license) to your account region's office headquarters.",
-        "notes_fr": "Le support vous demandera une lettre écrite demandant la suppresion du compte, dans cette lettre donnez vos details de compte ainsi qu'un justificatif d'identité.",
-        "notes_it": "Il servizio clienti richiede di spedire una lettera firmata al quartier generale della tua nazione confermando le tue intenzioni, allegando i dettagli del tuo account e una copia di un documento (passaporto, patente)",
-        "notes_pt_br": "A Assistência ao Cliente irá pedir que você envie uma carta assinada confirmando os seus desejos, os detalhes de sua conta e uma cópia autenticada de identificação (passaporte, carteira de motorista) para o escritório da região de sua conta.",
-        "notes_cat": "L'Assistencia al Client li demanarà que envii una carta firmada confirmant els seus desitjos, els detalls de la seva conta i una copia d'identificació legal (Passaport, DNI...).",
-        "notes_es": "La asistencia al cliente te pedirá que envíes una carta firmada confirmando tus deseos, los detalles de la cuenta y una copia de una identificación legal (Pasaporte, DNI...).",
-        "notes_pl": "Dział wsparcia wymaga wysłania pisemnego potwierdzenia chęci usunięcia konta, wraz ze szczegółami konta użytkownika oraz kopii jednego z dokumentów potwierdzających tożsamość (paszport, prawo jazdy).",
-        "domains": [
-            "battle.net"
+        "name": "Battle.net / Blizzard",
+        "url": "https://eu.battle.net/support/en/article/2659",
+        "difficulty": "medium",
```

```suggestion
        "difficulty": "medium",
```
In your experience, was there any chance that the request would be processed automatically, or did it always involved some kind of interaction with support?

---

My request is still pending, but they did not force me to send them a written letter (which IMHO is a big plus). The request itself only requires a few clicks now. But they might ask for an ID which then has to be uploaded. And I don't think that any correspondence with the customer support is required, because on their official site they say: "Customer Support is unable to manually delete Blizzard accounts.".

---

I'm OK with this being `medium` then. If you get answered and it requires more communication, let us know, so we can move it back to `hard`.

If they never answer it though and your account still exists, we can put it as `impossible`